appengine/flexible_python37_and_earlier/multiple_services/README.md

```diff
@@ -0,0 +1,67 @@
+# Python Google Cloud Microservices Example - API Gateway
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=appengine/flexible/multiple_services/README.md
+
+This example demonstrates how to deploy multiple python services to [App Engine flexible environment](https://cloud.google.com/appengine/docs/flexible/)
+
+## To Run Locally
+
+Use [virtualenv](https://virtualenv.pypa.io/en/stable/) to set up each
+service's environment and start the each service in a separate terminal.
+
+Open a terminal and start the first service:
+
+```Bash
+$ cd gateway-service
+$ virtualenv -p python3 env
```

Comment:
Ditto for `virtualenv`

Reply:
Done. This is a little bit tricky because we need to ask customers to start two virtual environments.